server/routes/creatingLicences/index.ts

```diff
@@ -49,6 +49,7 @@ export default function Index({
   communityService,
   prisonerService,
   ukBankHolidayFeedService,
+  conditionService,
```

Comment:
should move to service wrapper approach like in later verisons of template project